src/time_travel/time_machine_clock.py

```diff
@@ -0,0 +1,33 @@
+"""A single clock for all patchers."""
+
+
+class TimeMachineClock(object):
```

Comment:
It's A-Okay, I just didn't know that having both setter and getter makes `+=` work on it :)